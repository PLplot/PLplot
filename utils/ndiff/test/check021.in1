# This test file contains numbers in various formats that should all
# be recognized by ndiff

# Some numbers with nonsignificant underscore
314159_26535_89793_23846
3.14159_26535_89793_23846
3.14159_26535_89793_23846+0099
3.14159_26535_89793_23846+099
3.14159_26535_89793_23846+99
3.14159_26535_89793_23846-0099
3.14159_26535_89793_23846-99
3.14159_26535_89793_23846d+99
3.14159_26535_89793_23846D+99
3.14159_26535_89793_23846d-99
3.14159_26535_89793_23846D-99
3.14159_26535_89793_23846d99
3.14159_26535_89793_23846D99
3.14159_26535_89793_23846e+99
3.14159_26535_89793_23846E+99
3.14159_26535_89793_23846e-99
3.14159_26535_89793_23846E-99
3.14159_26535_89793_23846e99
3.14159_26535_89793_23846E99
3.14159_26535_89793_23846Q+99
3.14159_26535_89793_23846q+99
3.14159_26535_89793_23846Q-99
3.14159_26535_89793_23846q-99
3.14159_26535_89793_23846Q99
3.14159_26535_89793_23846q99

# The same numbers with underscores removed
314159265358979323846
3.14159265358979323846
3.14159265358979323846+0099
3.14159265358979323846+099
3.14159265358979323846+99
3.14159265358979323846-0099
3.14159265358979323846-99
3.14159265358979323846d+99
3.14159265358979323846D+99
3.14159265358979323846d-99
3.14159265358979323846D-99
3.14159265358979323846d99
3.14159265358979323846D99
3.14159265358979323846e+99
3.14159265358979323846E+99
3.14159265358979323846e-99
3.14159265358979323846E-99
3.14159265358979323846e99
3.14159265358979323846E99
3.14159265358979323846Q+99
3.14159265358979323846q+99
3.14159265358979323846Q-99
3.14159265358979323846q-99
3.14159265358979323846Q99
3.14159265358979323846q99

# Some numbers with nonsignificant underscore, and no digit
# before the decimal point
.14159_26535_89793_23846
.14159_26535_89793_23846+0099
.14159_26535_89793_23846+099
.14159_26535_89793_23846+99
.14159_26535_89793_23846-0099
.14159_26535_89793_23846-99
.14159_26535_89793_23846d+99
.14159_26535_89793_23846D+99
.14159_26535_89793_23846d-99
.14159_26535_89793_23846D-99
.14159_26535_89793_23846d99
.14159_26535_89793_23846D99
.14159_26535_89793_23846e+99
.14159_26535_89793_23846E+99
.14159_26535_89793_23846e-99
.14159_26535_89793_23846E-99
.14159_26535_89793_23846e99
.14159_26535_89793_23846E99
.14159_26535_89793_23846Q+99
.14159_26535_89793_23846q+99
.14159_26535_89793_23846Q-99
.14159_26535_89793_23846q-99
.14159_26535_89793_23846Q99
.14159_26535_89793_23846q99

# The same numbers with underscores removed, and no digit
# before the decimal point
.14159265358979323846
.14159265358979323846+0099
.14159265358979323846+099
.14159265358979323846+99
.14159265358979323846-0099
.14159265358979323846-99
.14159265358979323846d+99
.14159265358979323846D+99
.14159265358979323846d-99
.14159265358979323846D-99
.14159265358979323846d99
.14159265358979323846D99
.14159265358979323846e+99
.14159265358979323846E+99
.14159265358979323846e-99
.14159265358979323846E-99
.14159265358979323846e99
.14159265358979323846E99
.14159265358979323846Q+99
.14159265358979323846q+99
.14159265358979323846Q-99
.14159265358979323846q-99
.14159265358979323846Q99
.14159265358979323846q99

# Some numbers with nonsignificant underscore and no decimal point
314159_26535_89793_23846
314159_26535_89793_23846
314159_26535_89793_23846+0099
314159_26535_89793_23846+099
314159_26535_89793_23846+99
314159_26535_89793_23846-0099
314159_26535_89793_23846-99
314159_26535_89793_23846d+99
314159_26535_89793_23846D+99
314159_26535_89793_23846d-99
314159_26535_89793_23846D-99
314159_26535_89793_23846d99
314159_26535_89793_23846D99
314159_26535_89793_23846e+99
314159_26535_89793_23846E+99
314159_26535_89793_23846e-99
314159_26535_89793_23846E-99
314159_26535_89793_23846e99
314159_26535_89793_23846E99
314159_26535_89793_23846Q+99
314159_26535_89793_23846q+99
314159_26535_89793_23846Q-99
314159_26535_89793_23846q-99
314159_26535_89793_23846Q99
314159_26535_89793_23846q99

# The same numbers with underscores removed and no decimal point
314159265358979323846
314159265358979323846+0099
314159265358979323846+099
314159265358979323846+99
314159265358979323846-0099
314159265358979323846-99
314159265358979323846d+99
314159265358979323846D+99
314159265358979323846d-99
314159265358979323846D-99
314159265358979323846d99
314159265358979323846D99
314159265358979323846e+99
314159265358979323846E+99
314159265358979323846e-99
314159265358979323846E-99
314159265358979323846e99
314159265358979323846E99
314159265358979323846Q+99
314159265358979323846q+99
314159265358979323846Q-99
314159265358979323846q-99
314159265358979323846Q99
314159265358979323846q99

# Some short numbers
31
3.1
3.1+0099
3.1+099
3.1+99
3.1-0099
3.1-99
3.1d+99
3.1D+99
3.1d-99
3.1D-99
3.1d99
3.1D99
3.1e+99
3.1E+99
3.1e-99
3.1E-99
3.1e99
3.1E99
3.1Q+99
3.1q+99
3.1Q-99
3.1q-99
3.1Q99
3.1q99

# Some short numbers, with no digit before the decimal point
.1
.1+0099
.1+099
.1+99
.1-0099
.1-99
.1d+99
.1D+99
.1d-99
.1D-99
.1d99
.1D99
.1e+99
.1E+99
.1e-99
.1E-99
.1e99
.1E99
.1Q+99
.1q+99
.1Q-99
.1q-99
.1Q99
.1q99

# Some short numbers with no decimal point
31
31
31+0099
31+099
31+99
31-0099
31-99
31d+99
31D+99
31d-99
31D-99
31d99
31D99
31e+99
31E+99
31e-99
31E-99
31e99
31E99
31Q+99
31q+99
31Q-99
31q-99
31Q99
31q99
