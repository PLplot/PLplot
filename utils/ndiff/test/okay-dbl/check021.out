5c5
< 314159_26535_89793_23846
--- field 1	relative error 1.00e+05
> 314159_26535_89793
32c32
< 314159265358979323846
--- field 1	relative error 1.00e+05
> 3141592653589793
113c113
< 314159_26535_89793_23846
--- field 1	relative error 1.00e+05
> 314159_26535_89793
114c114
< 314159_26535_89793_23846
--- field 1	relative error 1.00e+05
> 314159_26535_89793
115c115
< 314159_26535_89793_23846+0099
--- field 1	relative error 1.00e+05
> 314159_26535_89793+0099
116c116
< 314159_26535_89793_23846+099
--- field 1	relative error 1.00e+05
> 314159_26535_89793+099
117c117
< 314159_26535_89793_23846+99
--- field 1	relative error 1.00e+05
> 314159_26535_89793+99
118c118
< 314159_26535_89793_23846-0099
--- field 1	relative error 1.00e+05
> 314159_26535_89793-0099
119c119
< 314159_26535_89793_23846-99
--- field 1	relative error 1.00e+05
> 314159_26535_89793-99
120c120
< 314159_26535_89793_23846d+99
--- field 1	relative error 1.00e+05
> 314159_26535_89793d+99
121c121
< 314159_26535_89793_23846D+99
--- field 1	relative error 1.00e+05
> 314159_26535_89793D+99
122c122
< 314159_26535_89793_23846d-99
--- field 1	relative error 1.00e+05
> 314159_26535_89793d-99
123c123
< 314159_26535_89793_23846D-99
--- field 1	relative error 1.00e+05
> 314159_26535_89793D-99
124c124
< 314159_26535_89793_23846d99
--- field 1	relative error 1.00e+05
> 314159_26535_89793d99
125c125
< 314159_26535_89793_23846D99
--- field 1	relative error 1.00e+05
> 314159_26535_89793D99
126c126
< 314159_26535_89793_23846e+99
--- field 1	relative error 1.00e+05
> 314159_26535_89793e+99
127c127
< 314159_26535_89793_23846E+99
--- field 1	relative error 1.00e+05
> 314159_26535_89793E+99
128c128
< 314159_26535_89793_23846e-99
--- field 1	relative error 1.00e+05
> 314159_26535_89793e-99
129c129
< 314159_26535_89793_23846E-99
--- field 1	relative error 1.00e+05
> 314159_26535_89793E-99
130c130
< 314159_26535_89793_23846e99
--- field 1	relative error 1.00e+05
> 314159_26535_89793e99
131c131
< 314159_26535_89793_23846E99
--- field 1	relative error 1.00e+05
> 314159_26535_89793E99
132c132
< 314159_26535_89793_23846Q+99
--- field 1	relative error 1.00e+05
> 314159_26535_89793Q+99
133c133
< 314159_26535_89793_23846q+99
--- field 1	relative error 1.00e+05
> 314159_26535_89793q+99
134c134
< 314159_26535_89793_23846Q-99
--- field 1	relative error 1.00e+05
> 314159_26535_89793Q-99
135c135
< 314159_26535_89793_23846q-99
--- field 1	relative error 1.00e+05
> 314159_26535_89793q-99
136c136
< 314159_26535_89793_23846Q99
--- field 1	relative error 1.00e+05
> 314159_26535_89793Q99
137c137
< 314159_26535_89793_23846q99
--- field 1	relative error 1.00e+05
> 314159_26535_89793q99
140c140
< 314159265358979323846
--- field 1	relative error 1.00e+05
> 3141592653589793
141c141
< 314159265358979323846+0099
--- field 1	relative error 1.00e+05
> 3141592653589793+0099
142c142
< 314159265358979323846+099
--- field 1	relative error 1.00e+05
> 3141592653589793+099
143c143
< 314159265358979323846+99
--- field 1	relative error 1.00e+05
> 3141592653589793+99
144c144
< 314159265358979323846-0099
--- field 1	relative error 1.00e+05
> 3141592653589793-0099
145c145
< 314159265358979323846-99
--- field 1	relative error 1.00e+05
> 3141592653589793-99
146c146
< 314159265358979323846d+99
--- field 1	relative error 1.00e+05
> 3141592653589793d+99
147c147
< 314159265358979323846D+99
--- field 1	relative error 1.00e+05
> 3141592653589793D+99
148c148
< 314159265358979323846d-99
--- field 1	relative error 1.00e+05
> 3141592653589793d-99
149c149
< 314159265358979323846D-99
--- field 1	relative error 1.00e+05
> 3141592653589793D-99
150c150
< 314159265358979323846d99
--- field 1	relative error 1.00e+05
> 3141592653589793d99
151c151
< 314159265358979323846D99
--- field 1	relative error 1.00e+05
> 3141592653589793D99
152c152
< 314159265358979323846e+99
--- field 1	relative error 1.00e+05
> 3141592653589793e+99
153c153
< 314159265358979323846E+99
--- field 1	relative error 1.00e+05
> 3141592653589793E+99
154c154
< 314159265358979323846e-99
--- field 1	relative error 1.00e+05
> 3141592653589793e-99
155c155
< 314159265358979323846E-99
--- field 1	relative error 1.00e+05
> 3141592653589793E-99
156c156
< 314159265358979323846e99
--- field 1	relative error 1.00e+05
> 3141592653589793e99
157c157
< 314159265358979323846E99
--- field 1	relative error 1.00e+05
> 3141592653589793E99
158c158
< 314159265358979323846Q+99
--- field 1	relative error 1.00e+05
> 3141592653589793Q+99
159c159
< 314159265358979323846q+99
--- field 1	relative error 1.00e+05
> 3141592653589793q+99
160c160
< 314159265358979323846Q-99
--- field 1	relative error 1.00e+05
> 3141592653589793Q-99
161c161
< 314159265358979323846q-99
--- field 1	relative error 1.00e+05
> 3141592653589793q-99
162c162
< 314159265358979323846Q99
--- field 1	relative error 1.00e+05
> 3141592653589793Q99
163c163
< 314159265358979323846q99
--- field 1	relative error 1.00e+05
> 3141592653589793q99
### Maximum relative error in matching lines = 1.61e-15 at line 64 field 1
